src/main/java/edu/ie3/datamodel/models/StandardUnits.java

```diff
@@ -89,6 +89,10 @@
   public static final Unit<ThermalConductance> THERMAL_TRANSMISSION = KILOWATT_PER_KELVIN;
   /** Depth of discharge */
   public static final Unit<Dimensionless> DOD = PERCENT;
+  /** Charge/Discharge Rate (constant power) */
+  public static final Unit<Dimensionless> CP_RATE = PU;
+  /** Charge/Discharge time at a given CP_Rate */
```

Comment:
What is meant with this parameter? The time to fully discharge (from maximum to minimum state of charge)? Possibly it is the same as suggested in my comment above.